website/docs/src/content/cli-reference/deploy.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: gensx deploy
+description: Deploy a workflow to GenSX Cloud
+---
+
+# gensx deploy
+
+The `gensx deploy` command uploads and deploys a workflow to GenSX Cloud, making it available for remote execution.
+
+## Usage
+
+```bash
+gensx deploy <file> [options]
+```
+
+## Arguments
+
+| Argument | Description                                                                                |
+| -------- | ------------------------------------------------------------------------------------------ |
+| `<file>` | File to deploy. This should be a TypeScript/JavaScript file that exports a GenSX Workflow. |
```

Comment:
Doesn't it have to be typescript so that we can detect the schema?

```suggestion
| `<file>` | File to deploy. This should be a TypeScript file that exports a GenSX Workflow. |
```

Reply:
Also - workflow should be lower case. General practice is that product names ("GenSX" and "GenSX Cloud" get capitalized but feature names do not.